src/main/kotlin/no/nav/k9/tjenester/saksbehandler/oppgave/OppgaveTjeneste.kt

```diff
@@ -451,7 +451,7 @@ class OppgaveTjeneste constructor(
     private suspend fun lagOppgaveDtoer(oppgaver: List<Oppgave>): OppgaverResultat {
         var ikkeTilgang = false
         val oppgaver = oppgaver.filter { oppgave ->
-            if (!pepClient.harTilgangTilLesSak(
+            if (!oppgave.harFagSaksNummer() || !pepClient.harTilgangTilLesSak(
                     fagsakNummer = oppgave.fagsakSaksnummer,
                     aktørid = oppgave.aktorId
                 )
```

Comment:
Kanskje lage en metode for ``harTilgangTilOppgave`` slik at du slipper å gjenta sjekken på fagsaknummer? Da blir det også enklere å legge til flere sjekker på oppgavetilgang senere.